hbase-server/src/main/java/org/apache/hadoop/hbase/HDFSBlocksDistribution.java

```diff
@@ -228,33 +228,49 @@ public long getUniqueBlocksTotalWeight() {
    * Implementations 'visit' hostAndWeight.
    */
   public interface Visitor {
-    float visit(final HostAndWeight hostAndWeight);
+    long visit(final HostAndWeight hostAndWeight);
```

Comment:
Can we not continue with `float`? Or is it that all instance var of `HostAndWeight` are long and hence, we want them as long here?

Reply:
Just thought we need fetch original value now, change it to long will more flexbility.
Thanks.